README.md

```diff
@@ -1,14 +1,15 @@
 # The Fuck [![Version][version-badge]][version-link] [![Build Status][travis-badge]][travis-link] [![Windows Build Status][appveyor-badge]][appveyor-link] [![Coverage][coverage-badge]][coverage-link] [![MIT License][license-badge]](LICENSE.md)
 
-Magnificent app which corrects your previous console command,
-inspired by a [@liamosaur](https://twitter.com/liamosaur/)
-[tweet](https://twitter.com/liamosaur/status/506975850596536320).
+*The Fuck* is a magnificent app inspired by a [@liamosaur](https://twitter.com/liamosaur/)
+[tweet](https://twitter.com/liamosaur/status/506975850596536320)
+that corrects errors in previous console commands.
```

Comment:
I think we should keep the comma between clauses. That is:

> *The Fuck* is a magnificent app inspired by a [@liamosaur](https://twitter.com/liamosaur/) [tweet](https://twitter.com/liamosaur/status/506975850596536320), that corrects errors in previous console commands.

instead of:

> *The Fuck* is a magnificent app inspired by a [@liamosaur](https://twitter.com/liamosaur/) [tweet](https://twitter.com/liamosaur/status/506975850596536320) that corrects errors in previous console commands.

to avoid suggesting that the tweet corrects errors :P